mqt/benchviewer/src/tests/test_backend.py

```diff
@@ -232,9 +232,18 @@ def test_prepareFormInput():
     assert backend.prepareFormInput(form_data) == expected_res
 
 
-def test_create_database():
+def test_read_mqtbench_all_zip():
+    assert backend.read_mqtbench_all_zip(skip_question=True)
+
+
+def test_handle_downloading_benchmarks():
+    backend.handle_downloading_benchmarks(
+        "v1.0.0", "mqt/benchviewer/static/files/qasm_output"
```

Comment:
as mentioned above, I am not yet a big fan of hardcoding the version for this function as this also implies that it has to be kept up to date for every release that changes the files.

Reply:
Version number is now read from the package itself and the corresponding version is automatically determined and downloaded